packages/go/go-helpers.ts

```diff
@@ -6,8 +6,9 @@ import { join } from 'path';
 import stringArgv from 'string-argv';
 import { debug } from '@vercel/build-utils';
 const versionMap = new Map([
-  ['1.16', '1.16'],
-  ['1.15', '1.15.8'],
+  ['1.17', '1.17.1'],
+  ['1.16', '1.16.8'],
+  ['1.15', '1.15.15'],
```

Comment:
```suggestion
  ['1.15', '1.15.8'],
```

Let's leave 1.15 alone since its considered "archived" according to https://golang.org/dl/#stable